src/frontend-scripts/components/section-right/Playerlist.jsx

```diff
@@ -1,693 +1,693 @@
+					<div className="ui basic modal playerlistinfo">
+						<div className="header">Lobby and player color info:</div>
+						<p>
+							Players in the lobby, general chat, and game chat are grey/white until they reach 50 games played. After that, they are known as "rainbow players"
+							because their color changes based on their stats. Rainbow players have access to play in special rainbow player only games.
+						</p>
+						<p>
+							The color of a rainbow player depends on their ELO, a type of matchmaking rating. The spectrum of colors goes from deep green as lowest ELO to
+							deep purple as highest ELO, passing through yellow and orange on its way.
+						</p>
+						<p>
+							Additionally, <span className="admin">Administrators</span> have a <span className="admin">red color</span> with a{' '}
+							<span className="admin-name">(A)</span> and are always at the top of the list.
+							<br />
+							<span className="cbell">Ed</span>
+							<span className="max">it</span>
+							<span className="moira">or</span>
+							<span className="thejuststopo">s</span>, placed at the top just below <span className="admin">Administrators</span>, have a range of special
+							colors to stand out, as well as a <span className="admin">(E)</span>.<br />
+							<span className="moderatorcolor">Moderators</span>, placed at the top below <span className="cbell">Ed</span>
+							<span className="max">it</span>
+							<span className="moira">or</span>
+							<span className="thejuststopo">s</span>, have a <span className="moderatorcolor">blue color</span> with a{' '}
+							<span className="moderatorcolor">(M)</span>.<br />
+							AEM <span className="veteran">Veterans</span> are retired senior moderators, and are given a <span className="veteran">teal</span> color.
+							<br />
+							Lastly, <span className="contributor">Contributors</span> get a <span className="contributor">special color</span> as well! Contribute code to
+							this open source project!.
+						</p>
+					</div>
+					{Object.keys(this.props.userList).length && (
+						<span>
+							<span>{this.props.userList.list.length}</span>
+							<i className="large user icon" title="Number of players logged in" />
+						</span>
+					)}
+				</div>
+				<Scrollbars renderThumbVertical={props => <div {...props} className="thumb-vertical" />}>
+					<div className="playerlist-body">
+						{this.props.userInfo.gameSettings && this.props.userInfo.gameSettings.disableAggregations ? this.renderLegacyPlayerlist() : this.renderPlayerlist()}
+					</div>
+				</Scrollbars>
+			</section>
+		);
+	}
+}
+
+Playerlist.defaultProps = {
+	userInfo: {},
+	userList: { list: [] },
+	socket: {}
+};
+
+Playerlist.propTypes = {
+	userInfo: PropTypes.object,
+	userList: PropTypes.object,
+	socket: PropTypes.object,
+	isUserClickable: PropTypes.bool,
+	fetchReplay: PropTypes.func
+};
+
+export default connect(mapStateToProps, mapDispatchToProps, mergeProps)(Playerlist);
```

Comment:
you diffed the entire file....?

Reply:
it was CRLF before, this PR just changed it to LF. 

I'll add the prettier rules and update it repo-wide later this week (after resolving most PRs to prevent conflicts) 👍